src/missing_tokens.py

```diff
@@ -28,67 +27,75 @@ def __init__(self, address: Address):
         token_contract = w3.eth.contract(
             address=Web3.toChecksumAddress(self.address),
             abi=ERC20_ABI,
-            ContractFactoryClass=Factory,
         )
-        self.symbol = token_contract.symbol()
-        self.decimals = token_contract.decimals()
+        self.symbol = token_contract.caller.symbol()
+        self.decimals = token_contract.caller.decimals()
 
     def to_str(self, version: DuneVersion):
         if version == DuneVersion.V1:
-            address_bytea = f"\\\\x{self.address[2:]}"
-            return f"{address_bytea}\t{self.symbol}\t{self.decimals}"
+            self.v1_string()
         if version == DuneVersion.V2:
-            return f"('{self.address.lower()}', '{self.symbol}', {self.decimals}),"
+            return self.v2_string()
         raise ValueError(f"Invalid DuneVersion {version}")
 
+    def v1_string(self) -> str:
+        address_bytea = f"\\\\x{self.address[2:]}"
+        return f"{address_bytea}\t{self.symbol}\t{self.decimals}"
+
+    def v2_string(self) -> str:
```

Comment:
In my opinion, these method names were hard to get based only on their name. I think they could be a little bit more descriptive and follow the reference of Clean Code: 

>Methods should have verb or verb phrase names like postPayment , deletePage, or save.

ref: https://www.oreilly.com/library/view/clean-code-a/9780136083238/

Reply:
Sure. I'll try and think of something, but also open to suggestions.